### DISCUSSION TITLE
Sheet Orders Problems

### DISCUSSION THREAD

**Discussion by shenshihao520** [jxlsteam/jxls#124] [Q&A]
Hello

  I got one excel template,it has two sheet,the first one was a mutiSheet export,the second one was a single sheet export,but when i finished the export,the sheet orders has changed,that single sheet came to the first position,and mutiSheet came to second,How can I fix the problem?
  Here is my mark-up,
mutiSheet: jx:area(lastCell="E22")  jx:each(items="pages", var="page", lastCell="E22", multisheet="sheetNames")
singleSheet: jx:area(lastCell="E5") jx:each(items="fund" var="f" lastCell="E3")

Best wishes

**Comment by SoltauFintel** [comment]
With the multisheet feature, the new sheets are always appended at the end. I don't currently have a working workspace, so I can't help at the moment. I guess you can use Apache POI commands to change the order of the sheets in the finished file.

**Reply by shenshihao520** [reply]
Can I understand that this is a bug？

**Reply by SoltauFintel** [reply]
"bug" is the wrong word. It would be an extension of the feature.

My company has the same requirement.
